docs/resources/ip_acl_lists.md

```diff
@@ -0,0 +1,40 @@
+# databricks_ip_access_lists Resource
+
+This resource allows you to create IP Access Lists in Databricks to control access to your workspace by IP. All IPs and CIDR ranges from each enabled list is put together applying type "BLACKLIST" first, then if still allowed type "WHITELIST" is checked.  Please see [IP Access List](https://docs.databricks.com/security/network/ip-access-list.html) for full feature documentation.
+
+-> **Note** The total number of IP addresses and CIDR scopes provided across all ACL Lists in a workspace can not exceed 1000.  Refer to the docs above for specifics.
+
+## Example Usage
+
+```hcl
+resource "databricks_ip_access_list" "allowed-list" {
+  label = "allow_in"
+  list_type = "WHITELIST"
+  ip_addresses = [
+    "1.2.3.0/24",
+    "1.2.5.0/24"
+  ]
+  depends_on = [<Replace with terraform workspace id syntax>]
```

Comment:
why do we need explicit `depends_on`? resource internally must retry until necessary state is reached (e.g. workspace being in a runnable state). this way we're having more smooth user journey, compared to creating azure databricks workspace with vnet injection :)